magpie/magpieadapter/__init__.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import os
+import sys
+
+# TODO Is the following block really useful?
+this_dir = os.path.abspath(os.path.dirname(__file__))
+sys.path.insert(0, this_dir)
+
+
+from twitcher.adapter.base import AdapterInterface
+from twitcher.owsproxy import owsproxy
+
+
+logger = logging.getLogger(__name__)
+
+
+__version__ = '0.1.1'
+
+
+class MagpieAdapter(AdapterInterface):
+
+    def servicestore_factory(self, registry, database=None, headers=None):
+        from magpie.magpieadapter.magpieservice import MagpieServiceStore
+        return MagpieServiceStore(registry=registry, headers=headers)
+
+    def owssecurity_factory(self, registry):
+        from magpie.magpieadapter.magpieowssecurity import MagpieOWSSecurity
+        # TODO For magpie we cannot store the servicestore object since the constructor need a header with token
+        # taken from the request... maybe we should check for that?!?
+        #return MagpieOWSSecurity(tokenstore_factory(registry), servicestore_factory(registry))
+        return MagpieOWSSecurity()
+
+    def configurator_factory(self, settings):
+        from pyramid.config import Configurator
+        from pyramid.authentication import AuthTktAuthenticationPolicy
+        from pyramid.authorization import ACLAuthorizationPolicy
+        from ziggurat_foundations.models import groupfinder
```

Comment:
use 
`from magpie.definitions.ziggurat_definitions import groupfinder` 
`from magpie.definitions.ziggurat_definitions import Configurator, ...`

I would also create a similar 'twitcher_definitions' file to allow import everything from a single call/location